pkg/webhook/podspec_dryrun.go

```diff
@@ -71,18 +68,18 @@ func validatePodSpec(ctx context.Context, ps v1.RevisionSpec, namespace string)
 
 // dryRunPodSpec makes a dry-run call to k8s to validate the podspec
 func dryRunPodSpec(ctx context.Context, pod *corev1.Pod) *apis.FieldError {
-	logger := logging.FromContext(ctx)
 	client := kubeclient.Get(ctx)
 	pods := client.CoreV1().Pods(pod.GetNamespace())
 
 	options := metav1.CreateOptions{DryRun: []string{metav1.DryRunAll}}
 	if _, err := pods.CreateWithOptions(ctx, pod, options); err != nil {
 		// Ignore failures for implementations that don't support dry-run.
 		// This likely means there are other webhooks on the PodSpec Create action which do not declare sideEffects:none
-		if strings.Contains(err.Error(), "does not support dry run") {
+		// TODO(whaught): reinstate this after testing.
+		/*if strings.Contains(err.Error(), "does not support dry run") {
 			logger.Warnw("dry run validation failed, a webhook did not support dry-run", zap.Error(err))
 			return nil
-		}
+		}*/
```

Comment:
do we need  it or still wip here?

Reply:
This is some wip while I'm testing it locally